go-app-ussd_popi_user_data.js

```diff
@@ -375,39 +375,67 @@ go.app = function() {
                     }
                 },
                 next: function(content) {
-                    // TODO: Update ID on identitystore
-                    return 'state_updated';
+                    return 'state_create_identification_change';
                 }
             });
         });
 
-        self.add('state_change_passport', function(name) {
+        self.add('state_passport_origin', function(name) {
             return new ChoiceState(name, {
                 question: $('What is the country of origin of the passport?'),
                 choices: [
-                    new Choice('state_change_passport_zim', $('Zimbabwe')),
-                    new Choice('state_change_passport_moz', $('Mozambique')),
-                    new Choice('state_change_passport_mal', $('Malawi')),
-                    new Choice('state_change_passport_ng', $('Nigeria')),
-                    new Choice('state_change_passport_drc', $('DRC')),
-                    new Choice('state_change_passport_som', $('Somalia')),
-                    new Choice('state_change_passport_oth', $('Other'))
+                    new Choice('zw', $('Zimbabwe')),
+                    new Choice('mz', $('Mozambique')),
+                    new Choice('mw', $('Malawi')),
+                    new Choice('ng', $('Nigeria')),
+                    new Choice('cd', $('DRC')),
+                    new Choice('so', $('Somalia')),
+                    new Choice('other', $('Other'))
```

Comment:
In the current ndoh_hub setup, we have a different list of countries: https://github.com/praekeltfoundation/ndoh-hub/blob/develop/ndoh_hub/utils.py#L79 . I'm not sure whether that list of countries needs to change, or whether we need to change the list of countries in the hub app.